packages/tasks/src/deposit.ts

```diff
@@ -0,0 +1,89 @@
+import confirm from "@inquirer/confirm";
+import ERC20_ABI from "@openzeppelin/contracts/build/contracts/ERC20.json";
+import { task } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { ZetaChainClient } from "../../client/src";
+
+const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
+  const { ethers } = hre as any;
+  const [signer] = await ethers.getSigners();
+
+  const client = new ZetaChainClient({ network: "testnet", signer });
+
+  const { amount, erc20 } = args;
+
+  let message;
+  if (args.message) {
+    try {
+      const m = JSON.parse(args.message);
```

Comment:
Can we rename to something less ambiguous?

Reply:
a35745f